master/getting-started/mesos/installation/integration.md

```diff
@@ -54,6 +54,24 @@ Calico runs as a Docker container on each host. The `calicoctl` command line too
    chmod +x $NETWORK_CNI_PLUGINS_DIR/calico-ipam
```

Comment:
Can we simplify the opening para of this guide - I don't really understand it:  "This guide explains the components necessary to install Calico on Mesos for integrating with custom configuration management."

Shouldn't it just say "This guide explains how to integrate Calico networking and policy on an existing Mesos cluster"   ... or something like that?